scripts/deployment/deploy.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import dataclasses
+import argparse
+import logging
+import os
+import psycopg2
+import copy
+
+
+@dataclasses.dataclass
+class PostgresDBConn:
+    """This dataclass contains all information related to making a connection to Postgres DB."""
+    username: str
+    password: str
+    host: str
+    database: str
+    port: int
+
+
+def parse_args() -> argparse.Namespace:
+    """CLI args parsing function."""
+    parser = argparse.ArgumentParser(
+        description="Deploys structures to DB. (script version: 1.0)",
```

Comment:
not sure why information about the script version is needed but I would just not mention it

Reply:
It's a stupid script. I would like to have some version somewhere. And if I am having it, why not print it with help?

Reply:
I'll leave it up to you, but versioning individual scripts (for which purpose?) seems odd, especially since we are versioning the whole repo, i.e. we are not versioning individual files or modules, just the whole codebase 🤷‍♂️